web-ng/src/app/components/layer-dialog/layer-dialog.component.ts

```diff
@@ -173,13 +169,14 @@ export class LayerDialogComponent implements OnDestroy {
     const fields = this.layerService.convertFieldsListToMap(this.fields);
     const formId = this.form?.id;
     const forms = this.layerService.createForm(formId, fields);
-    let contributions: string[] = [];
-    if (this.contributorsCanAddPoint && this.contributorsCanAddPolygon) {
-      contributions = ['points', 'polygons'];
-    } else if (this.contributorsCanAddPolygon) {
-      contributions = ['polygons'];
-    } else if (this.contributorsCanAddPoint) {
-      contributions = ['points'];
+    const allowedFeatureTypes: string[] = [];
+    if (this.contributorsCanAddPoints && this.contributorsCanAddPolygons) {
```

Comment:
These conditional blocks are overlapping/redundant - please see code in previous comment for solution.

Reply:
updated.